### ISSUE TITLE
Seek tracks

### ISSUE THREAD

**Issue by gardar** [tizonia/tizonia-openmax-il#438]
There does not seem to be any seek support currently (or at least no key bindings for seeking).

I mostly use tizonia to listen to long tracks (1-2 hour long tracks) and it would be good to be able to seek into tracks, especially with issue #437 where I can't resume playback after pausing.

**Comment by tizonia** [comment]
That's correct, seek is not implemented yet. This is a reflection of how I listen to music more than anything, but with things like podcasts and the like, Tizonia is really in need of the seek functionality.

I can't promise when this would be ready, but this is definitely higher now in my todo list.

**Comment by gardar** [comment]
Excellent!

While we are on the topic, it would also be great to see the time position of the currently playing item, currently I can see the total duration but not current position.

**Comment by tizonia** [comment]
I agree. The absence of the time position has been a forgivable issue until now because there was no seek option.

**Comment by kingo55** [comment]
If you are going through your playlist and skipping tracks, it would be useful if Tizonia maintained the position of the song you left when you skip to the next track. So if you return to the previous song in the playlist it will resume playing from where you left off...

**Comment by comod** [comment]
I am new to this Project. I was searching for a simple music Player for soundcloud and some streams. For soundcloud i miss definitly the seek functionality due i listen often to longer sets instead of single tracks. Also i miss the abitility to use my global keyboard fn keys for play, pause, for and Backwards. Also it would be useful to like or even leave a comment for soundcloud. But to keep in the line here the seek function is really needed!
